README.md

```diff
@@ -409,6 +419,22 @@ exports.handler = (event, context, callback) => {
 };
 ```
 
+## HTTP Exception handling
+If the logger did not receive a successful response from the server when tried to send a log,
+it will retain the logs in its buffer and will try to flush it with the next request.
+The flush limit of the line that comes after a failed one will be reset to the backoff period.
+The failed messages retention byte size is limited. The limit and back off
+period are configurable via options:
+
```

Comment:
Let's use: `If the logger does not receive a successful response from the server, it retains the logs in a buffer and will retry with the next request. The size of the retry buffer that saves logs that failed to send and the  retry timeout are configurable via:`